testing/features/input.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Input
+  As a non-technical person at Citizens Advice
+  I want to know the full specification of what an “Input” component can / should do
+  So I can check whether it is being used correctly
+  And I can check if is buggy
+
+  Rule: Basic
+    Background:
+      Given I have a Basic Input component on page
+
+    Scenario: Input is on the page and I can type on it
```

Comment:
Lets re-word this to something like.

Input form is on the page without any assistance. We can also re-word the other ones accordingly.